### ISSUE TITLE
Wiki links don't work in VS Code preview mode

### ISSUE THREAD

**Issue by OG-Slips** [foambubble/foam-template#9]
When viewing notes in preview mode the [[wiki-links]] aren't clickable. They are, however, clickable when in editing mode. I might be in the minority (or missing something), but as someone who would be using this entirely offline, this breaks the ability to view and freely peruse the linked notes without switching into viewing the raw markdown.

**Comment by todrobbins** [comment]
@OG-Slips I'm able to browse via the editor or the preview:

![foam](https://user-images.githubusercontent.com/158590/86543598-5b7a1a00-bedd-11ea-8203-c5eb6c22cf82.gif)

Which version of VS Code are you running? Also, make sure you don't skip this step:

> When prompted to install recommended extensions, click **Install all** (or **Show Recommendations** if you want to review and install them one by one)

**Comment by OG-Slips** [comment]
I doubled checked and all the recommended extensions are installed and enabled. I'm currently using VS Code 1.46.1.

**Comment by jevakallio** [comment]
@OG-Slips thanks for reporting! Are you seeing link reference definition generated at the end of the file?

See https://foambubble.github.io/foam/wiki-links#markdown-compatibility

If not, ensure you're on Foam for VS Code 0.1.7 or later.

In order for it to run automatically, you need a .VSCode/foam.json. If this is missing, you can run the "update wiki links" command from the command palette.

If the issue persists, can you share a .md file (or even better, a foam workspace generated from foam-template) that reproduces this issue.

**Comment by OG-Slips** [comment]
I actually just seemed to figure it out after noticing the example links were all lowercase. I was writing the links with mixed case ( such as [[Test-File]] instead of [[test-file]]. Before changing, no references were generated and updating links didn't do anything. I changed my links to all lowercase and updated the markdown references, which caused the links to work as intended. 

I could have easily missed this in the documentation somewhere (double checked to make sure I wasn't just being oblivious), but if it's not there might be worth mentioning links are case-sensitive. 

Thank you both for helping me troubleshoot. I'll go ahead and mark this as closed.
